app/connectors/httpParsers/IvdHttpParser.scala

```diff
@@ -68,24 +69,24 @@ object IvdHttpParser {
     }
   }
 
-  implicit object UpdateResponseReads extends HttpReads[HttpPostResult[UpdateResponse]] {
+  implicit object UpdateResponseReads extends HttpReads[Either[UpdateCaseError, UpdateCaseResponse]] {
 
     private val logger = Logger("application." + getClass.getCanonicalName)
 
-    override def read(method: String, url: String, response: HttpResponse): HttpPostResult[UpdateResponse] = {
+    override def read(method: String, url: String, response: HttpResponse): Either[UpdateCaseError, UpdateCaseResponse] = {
 
       response.status match {
-        case Status.OK =>
-          response.json.validate[UpdateResponse].fold(
-            invalid => {
-              logger.error("Failed to validate JSON with errors: " + invalid)
-              Left(ErrorModel(Status.INTERNAL_SERVER_ERROR, "Invalid Json returned from IVD Update Case"))
-            },
-            valid => Right(valid)
-          )
+        case Status.OK => Right(UpdateCaseResponse())
```

Comment:
I think I would like to pull the case id out of the response from Pega so that when I audit it I have a record of what was sent and received.